src/utils/networkUtils.ts

```diff
@@ -71,12 +85,20 @@ export async function getEthereumNetwork() {
 	}
 }
 
-export const getTransactionPrice = (gasPrice, gasLimit, ethPrice) => {
+export const getTransactionPrice = (
+	gasPrice: number | null,
+	gasLimit: number | null,
+	ethPrice: number | null
+) => {
 	if (!gasPrice || !gasLimit || !ethPrice) return 0;
```

Comment:
should prob say gasPrice == null || gasLimit == null || ethPrice == null since it is either going to be null or a number. What if gasPrice is 0 though?

Reply:
I actually didn't want to touch the actual logic since it worked, just added the types here. Falsy values should be fine here! `0`,`null`,etc...all good - and if one of them is falsy, it should return `0`. Because, the formula is 
```
(gasPrice * ethPrice * gasLimit) / GWEI_UNIT
```